lms/templates/instructor/instructor_dashboard_2/certificate-invalidation.underscore

```diff
@@ -3,9 +3,16 @@
 </p>
 
 <div class="add-certificate-invalidation">
-    <input class='student-username-or-email' id="certificate-invalidation-user" type="text" placeholder="<%- gettext('Username or email address') %>">
-    <textarea class='notes-field' id="certificate-invalidation-notes" rows="10" placeholder="<%- gettext('Add notes about this learner') %>"></textarea>
+    <div>
+        <label for="certificate-invalidation-user"><%- gettext('Username or email address') %></label>
```

Comment:
I would like to see some sort of technical debt story created about the issues in this HTML. I don't necessarily think you would need to do it-- it might make more sense for the original author(s)/owners of the HTML to do it.